GermanFilter/sections/specific.txt

```diff
@@ -86,6 +86,9 @@ pcwelt.de##div[data-ga-event-category="billiger_de_box_right_column"]
 pcwelt.de##.marketplace
 flashscore.de##.boxOverContent__banner
 filmstarts.de##.main-banner
+la-palma24.info##.dvtopgadogle
+la-palma24.info##.prmprinone
+la-palma24.info##.dos-ads-colums-r
```

Comment:
bad rule
![image](https://user-images.githubusercontent.com/8361299/115415645-c0e68f80-a1ff-11eb-83b9-4110eb06d1d1.png)

Reply:
@Alex-302, i found out that there is already `la-palma24.info#?#main > div.dvbloqbasic:has(> div[title="Anzeige"])` that i added some time ago and that it only apply from userfilter for some reason.